nix/devShells/default.nix

```diff
@@ -0,0 +1,32 @@
+{
+  callPackage,
+  checks,
+  craneLib,
+  binPkgs,
+  topiaryPkgs,
+}:
+
+{
+  default = callPackage ./devShell.nix {
+    inherit craneLib binPkgs checks;
+  };
+
+  light = callPackage ./devShell.nix {
```

Comment:
Name suggestion (feel free to ignore): `minimal` instead of `light`?

Reply:
`minimal` works for me. I'm not sure if anyone uses the non-default devshells.

Reply:
Done in b853c7827d056b26ae69f52e267f8549ca2a2a9c.